db/app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
```

Comment:
This is used to allow our servers to parse incoming JSON from the request body - at this point we have no such functionality so we should get rid of this. We want to keep things clean and strictly 'as necessary'